tools/theme/components/badge.ts

```diff
@@ -0,0 +1,49 @@
+import { ComponentSingleStyleConfig } from '@chakra-ui/react'; // Single component style config
+
+// Badge style config
+const Badge: ComponentSingleStyleConfig = {
+  // Base style
+  baseStyle: {
+    display: 'flex',
+    alignItems: 'center',
+    borderRadius: '23px',
+    textTransform: 'inherit',
+    fontWeight: 'normal',
+    fontSize: '16px',
+    height: '32px',
+    paddingX: '1rem',
+    paddingY: '4px',
+  },
+
+  // Variants (completed, inProgress, pending, rejected, expiring, expired, active, inactive)
+  variants: {
```

Comment:
Since we're just returning props objects here, we can replace all of the functions with plain objects, like so:
```javascript
completed: {
  bg: '#EEEEEE',
}
```